src/store/users/actions.ts

```diff
@@ -20,6 +20,7 @@ export const fetchCollaboratorsAction = createAsyncThunk<APICollection<Collabora
         // Wrap the client request in a promise to allow it to be returned and cancelled
         return new Promise<APICollection<Collaborator>>((resolve, reject) => {
             client.getFileCollaborators(fileId, resolve, reject, {
+                filter_term: searchStr, // eslint-disable-line @typescript-eslint/camelcase
```

Comment:
Does this need a default value? Is an empty string handled properly?

Reply:
This action will not be called if it's empty string. The check is in `ReplyField`.